gitlab-runner-17.1.yaml

```
@@ -33,7 +39,7 @@ pipeline:
       ldflags: -s -w -X gitlab.com/gitlab-org/gitlab-runner/common.NAME=${{package.name}} -X gitlab.com/gitlab-org/gitlab-runner/common.VERSION=v${{package.version}}
 
 subpackages:
-  - name: gitlab-runner-helper
+  - name: gitlab-runner-helper-${{vars.short-package-version}}
```

```suggestion
  - name: ${{package.name}}-helper
```

would this work?  i.e. so that the subpackage includes the version stream from the package name?  Then we wont need the var-transforms?